pkg/config/plugins/runtime/k8s/config.go

```diff
@@ -56,6 +56,10 @@ func DefaultKubernetesRuntimeConfig() *KubernetesRuntimeConfig {
 							Memory: "512Mi",
 						},
 					},
+					LifeCycle: LifeCycleEvents{
+						// Experimental feature, disabled by default
+						SidecarInitTimeout: -1,
```

Comment:
can we use 0 as the default? We treat 0 and -1 the same.

Reply:
Sure, originally I thought that 0 would be for an infinite timeout instead of disabled. I'll change it